deployment/ansible/deploy.yml

```diff
@@ -65,6 +65,13 @@
       when: local_package is defined
       tags: [package]
 
+    - name: "Make sure chain path ( {{ config.chain.db_path }} ) exists"
+      file:
+        path: "{{ config.chain.db_path }}"
```

Comment:
@lucafavatella We missed the {{ project_root }} prefix here causing to create the directory in wrong parent

Reply:
https://github.com/aeternity/epoch/pull/415